tablecodec/tablecodec.go

```diff
@@ -623,6 +643,14 @@ func GenTableIndexPrefix(tableID int64) kv.Key {
 	return appendTableIndexPrefix(buf, tableID)
 }
 
+// IsUntouchedIndexKValue uses to check whether the key is index key, and the value is untouched,
+// since the untouched index key/value is no need to commit.
+func IsUntouchedIndexKValue(k, v []byte) bool {
+	vLen := len(v)
+	return (len(k) > 11 && k[0] == 't' && k[9] == '_' && k[10] == 'i') &&
```

Comment:
It can be simplified to
```go
vLen := len(v)
return (len(k) > 11 && k[0] == 't' && k[10] == 'i') && (vLen > 0 && v[vLen-1] == '1')
```
And this function is very fast, less than 1ns to execute.
We don't need to maintain a `HasUntouchedIndex` state.

Reply:
done.